procServUtils/manage.py

```diff
@@ -139,6 +144,8 @@ def addproc(conf, args):
         if args.username: F.write("user = %s\n"%args.username)
         if args.group: F.write("group = %s\n"%args.group)
         if args.port: F.write("port = %s\n"%args.port)
+        if args.environment: F.write("environment = %s\n"%' '.join(args.environment))
```

Comment:
Some escaping would be a good idea at some point.  I suspect it should be here.  From `man systemd.exec` the end result should be something like:

> Environment="VAR1=word1 word2" VAR2=word3 "VAR3=$word 5 6"

Quoting for spaces is probably enough.  I can't find a quick answer to how to include literal `"` in variable values.

Reply:
Thanks for the feedback! 
If I understand correctly the problem arises when using keys with multiple values, which must be quoted. To fix all the possible cases I just added quotes to all the elements, even if they have a single value.